app/views/students/index.html.erb

```diff
@@ -1,6 +1,6 @@
 <h1>Here are the students</h1>
-<% @students.each do |student| %>
+<% @student.each do |student| %>
   <div>
-    <%= link_to student, students_path(student) %> - <%= student.house %>
+    <%= link_to student.name, student %>
+  </div>
 <% end %>
```

Comment:
Here you want to use `@students`  on line 2
You'd also want to use `  <div><%= link_to student.name, student_path(student) %> - <%= student.house.name %></div>` on line 4